Examples/SimpleMacApp/SimpleMacApp/CKMSampleTableComponent.mm

```diff
@@ -0,0 +1,169 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import "CKMSampleTableComponent.h"
+
+#import <ComponentKit/CKComponentSubclass.h>
+#import <ComponentKit/CKComponentInternal.h>
+#import <ComponentKit/CKNSTableViewDataSource.h>
+#import <ComponentKit/CKTransactionalComponentDataSourceChangeset.h>
+
+@interface CKMSampleTableComponent () {
+  @package
+  CKComponent *_table;
+  Class _cellProviderClass;
+}
+
+@property (nonatomic, copy) NSArray *model;
```

Comment:
Why this is RW? And nonatomic? Components are spinning in BG thread, and I think this should be readonly (maybe worth exposing as method `- (NSArray *)model[s?]` instead of property)